src/compile/compile.ts

```diff
@@ -59,17 +59,26 @@ function assemble(model: Model, topLevelProperties: TopLevelProperties) {
   // autoResize has to be put under autosize
   const {autoResize, ...topLevelProps} = topLevelProperties;
 
+  const encode = model.assembleParentGroupProperties();
+  let width;
```

Comment:
Looking at this again, maybe it's worth commenting why we don't need width/height here, but need it when we call `assembleParentGroupProperties` in other places?